application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -41,12 +45,24 @@ public TagsCommand(TagSystem tagSystem) {
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
-        // TODO A list might be better than comma separated, which is hard to read
+
+        ArrayList<String> list = new ArrayList<>(tagSystem.getAllIds());
+
+        if (list.size() > 200) {
+
+            Logger logger = Logger.getLogger(TagsCommand.class.getName());
```

Comment:
You should initialize the logger as static field during startup and not like that, see the other classes.